examples/string-sum/tests/test_.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from string_sum import sum_as_string
+
+def test_sum():
+    a, b = 12, 42
+
+    added = sum_as_string(a, b)
+    assert added == "54"
+
+def test_err1():
+    a, b = "abc", 42
+
+    with pytest.raises(TypeError, match = "sum_as_string expected an int for positional argument 1") as e:
+        sum_as_string(a, b)
+        
+
+def test_err2():
+    a, b = 0, {}
+
+    with pytest.raises(TypeError, match = "sum_as_string expected an int for positional argument 2") as e:
+        sum_as_string(a, b)
+        
+
+def test_overflow1():
+    a, b = 0, 1 << 43
+
+    with pytest.raises(OverflowError, match = "Python int too large to convert to C long") as e:
+        sum_as_string(a, b)
+
+def test_overflow2():
+    a, b = 1 << 30, 1 << 30
+
+    with pytest.raises(OverflowError, match = "arguments too large to add") as e:
+        sum_as_string(a, b)
```

Comment:
Did you test on a 32-bit Windows Python interpreter by any chance? Think you'll need to shift past 64 bits to overflow on most platforms 😄

Reply:
It's 32 bits on 64-bit Windows too 😆

Reply:
TIL 😂